dictionaries/CallMap.php

```diff
@@ -7693,256 +7693,314 @@
 'MongoDB::setReadPreference' => ['bool', 'read_preference'=>'string', 'tags='=>'array'],
 'MongoDB::setSlaveOkay' => ['bool', 'ok='=>'bool'],
 'MongoDB::setWriteConcern' => ['bool', 'w'=>'mixed', 'wtimeout='=>'int'],
-'MongoDB\BSON\Binary::__construct' => ['void', 'data'=>'string', 'type'=>'int'],
-'MongoDB\BSON\Binary::__toString' => ['string'],
+'MongoDB\BSON\Binary::__construct' => ['void', 'data' => 'string', 'type' => 'int'],
 'MongoDB\BSON\Binary::getData' => ['string'],
 'MongoDB\BSON\Binary::getType' => ['int'],
-'MongoDB\BSON\binary::jsonSerialize' => ['mixed'],
-'MongoDB\BSON\binary::serialize' => ['string'],
-'MongoDB\BSON\binary::unserialize' => ['void', 'serialized'=>'string'],
-'MongoDB\BSON\binaryinterface::__toString' => ['string'],
-'MongoDB\BSON\binaryinterface::getData' => ['string'],
-'MongoDB\BSON\binaryinterface::getType' => ['int'],
-'MongoDB\BSON\dbpointer::__construct' => ['void'],
-'MongoDB\BSON\dbpointer::__toString' => ['string'],
-'MongoDB\BSON\dbpointer::jsonSerialize' => ['mixed'],
-'MongoDB\BSON\dbpointer::serialize' => ['string'],
-'MongoDB\BSON\dbpointer::unserialize' => ['void', 'serialized'=>'string'],
-'MongoDB\BSON\Decimal128::__construct' => ['void', 'value='=>'string'],
+'MongoDB\BSON\Binary::__toString' => ['string'],
+'MongoDB\BSON\Binary::serialize' => ['string'],
+'MongoDB\BSON\Binary::unserialize' => ['void', 'serialized' => ''],
```

Comment:
serialized should have a type here, either mixed or more narrow

Reply:
Ah, thanks. Added the correct `string` type for all `unserialize` methods.

Reply:
I don't see a new commit on the branch? Forgot to push?

Reply:
Whopsie, yes. I pushed the commit now.

Reply:
Well, I have bad news then, I still don't see anything

Reply:
nvm, I was mistaken apparently